README.md

```diff
@@ -123,15 +129,16 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
+If there is at least one Lord and no active Uprising nail currently exists, Lads may call "Uprising" at the start of their turn before flipping the hammer. If they do, they place a single Uprising nail somewhere in the stump and hammer it to the same starting depth as all other nails. Uprising rules now apply to the duration of this turn.
 
-The active player forfeits the ability to give out drinks during an Uprising. However, they still must drink if they miss, regrip, drop the hammer, or are penalized in some other way.
+As long as there is an active Uprising nail and the current player is a Lad, they may call "Uprising" before flipping their hammer. If they do, then the turn officially counts as an Uprising and Uprising rules apply for the rest of the turn. 
 
-If the Uprising counter reaches 3 all Lords must immediately drink. If the Uprising counter reaches 7 a Coup occurs all Lords are immediately demoted to Lads and must drive a new starting nail into the outer ring of the stump. When a Coup occurs the Uprising counter is reset to 0.
+During an Uprising, the active player must perform a double hammer flip followed by one strike on the Uprising nail without hesitation (note that they do not get two strikes like they normally would). If this strikes completely hammers the Uprising nail into the stump, all Lords are immediately demoted to Lad and must begin a new starting nail. If the active player already has their starting nail hammered in (if the amount of Lords was previously the maximum) they then instantly ascend to Lord. The Uprising nail is no now longer active and a new one may be started by future Lads.
 
-If a player would normally become a Lord upon performing their Uprising strike, they do not become a Lord until any Uprising effects for this turn resolve. I.e., If the Uprising counter is at 6 and a player performs a successful Uprising but in the process also completely hammers their starting nail into the stump, all existing Lords would be demoted and then the active player would be promoted to Lord. A player may not give out drinks during an Uprising even if they become Lord.
+A Lad may call uprising and then perform a triple or even higher order flip in which case they may take multiple strikes to the Uprising nail or any other valid targets. However, the first strike must hit the center nail. The amount of strikes a player gets in this case is n - 1 (including the initial Uprising strike) where n is the number of flips.
 
+If a Lad hits the Uprising nail without performing the mandatory double flip, the Uprising nail is reset.
 
+If that player hits a nail during an Uprising, they may give out a drink to a Lord during the Drink phase. This is the only situation in which a Lad can give a Lord a drink without penalty.
 
+If a Lord strikes an Uprising nail they are instantly demoted to Lad unless that strike finishes the nail. In that case the Uprising is complete and all other Lords are demoted (they remain a Lord).
```

Comment:
We never tested this rule, so maybe make a note about trying it out or something.

Reply:
Do the same Uprising rules apply to the Lord's strike? (i.e. Declare "Uprising", 2 flips)

Reply:
I can move this into a separate PR, but this addresses some open issues we had. For instance there was no real penalty for a lord purposefully bending the uprising nail and also no recourse if someone hit the uprising nail without doing the double flip.

As it's written in this PR the Lord doesn't have to do anything special they just have to hit the nail, but they get demoted unless they finish it off. Instead how about they have to call "Uprising", double flip, and sink the uprising nail? not successfully finishing the nail or hitting it without calling it/double flipping would still be a demotion